docs/topics/review-existing-rules.adoc

```diff
@@ -12,4 +12,20 @@ You can fork and clone the {ProductShortName} XML rules on your local machine.
 
 Rules are grouped by target platform and function. When you create a new rule, it is helpful to find a rule that is similar to the one you need and use it as a starting template.
 
-New rules are continually added, so it is a good idea to check back frequently to review the updates.
+New XML rules are occasionally added, so it is a good idea to check back frequently to review the updates.
+
+NOTE
+.YAML rules
+====
+.Existing {ProductShortName} rules `.yaml` files
+
+XML rules are in the process of being replaced with newer YAML rules. 
+
+Most of the `.xml` rules files have been converted to comparable `.yaml` rule files. 
```

Comment:
💡 RedHat.PassiveVoice 3(36-50) 'been converted' is passive voice. In general, use active voice. Consult the style guide for acceptable use of passive voice.